src/api/app/models/review.rb

```diff
@@ -94,23 +97,31 @@ def check_initial
       errors.add(:unknown, 'no reviewer defined')
     end
 
-    if by_user && !User.find_by_login(by_user)
+    if by_user
```

Comment:
Do you think this is more readable? 
If you prefer I can do it but actually I think this is more complicated and harder to read!